docker/muranoservice/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:16.04
```

Comment:
I mean, I want that the Dockerfile starts with two lines in which you write how to create the image and how to launch the container. Just the docker commands.